README.md

```diff
@@ -8,220 +8,10 @@
 
 Mapbox Navigation gives you all the tools you need to add turn-by-turn navigation to your apps.
 
-Get up and running in a few minutes with our drop-in turn-by-turn navigation `NavigationViewController`, or build a completely custom turn-by-turn navigation app with our core components for routing and navigation.
-
-## Features
-
-- Drop-in turn-by-turn navigation UI
-- Automotive, cycling, and walking directions
-- Traffic avoidance
-- Maneuver announcements
-- Text instructions
-- Text to speech support via AVSpeechSynthesizer or Amazon Polly
-- Automatic rerouting
-- Snap to route
```

Comment:
We should keep a list of key features in the readme. Someone who happens upon this repository first may not discover or bother to visit the [product homepage](https://www.mapbox.com/navigation-sdk/) to learn why this SDK is best for them.